src/index.js

```diff
@@ -1,22 +1,17 @@
 import './style.css';
 import {
-  addScore, createScore, resetInput, getDisplay,
+  createGame, getPosts, resetInput,
 } from './functions';
 
 const submit = document.querySelector('.subBtn');
-const name = document.querySelector('.name');
-const score = document.querySelector('.score');
+const refBtn = document.querySelector('.refBtn');
 
 submit.addEventListener('click', (e) => {
   e.preventDefault();
-
-  const newScore = addScore(
-    name.value,
-    score.value,
-  );
-
-  createScore(newScore);
+  createGame();
   resetInput();
 });
 
-getDisplay();
+refBtn.addEventListener('click', () => {
+  getPosts();
+});
```

Comment:
- [ ] There is an issue with your refresh button, every time I click it it will add all the scores again to the page, I will recommend you to delete all the content and then add the scores again in every refresh 🐱
    - On the image below I clicked the button 2 times, you can see that the scores are repeated.
    ![imagen](https://user-images.githubusercontent.com/60789317/141355892-bf6c5818-66ac-4849-b93e-ed8c323034f2.png)